tools/cbmc/patches/0001-Remove-static-storage-class-from-entry-points.patch

```diff
@@ -7,15 +7,15 @@ Many of the entry points we wish to test are marked as being static.
 This commit removes the static keyword from all entry points that we
 test.
 ---
- lib/FreeRTOS-Plus-TCP/source/FreeRTOS_DHCP.c    |  6 +++---
- lib/FreeRTOS-Plus-TCP/source/FreeRTOS_DNS.c     | 14 +++++++-------
- lib/FreeRTOS-Plus-TCP/source/FreeRTOS_TCP_WIN.c |  2 +-
+ modules/libraries/standard/freertos_plus_tcp/source/FreeRTOS_DHCP.c    |  6 +++---
```

Comment:
we don't have "modules" anymore. The top directories are libraries and freertos_kernel

Reply:
Thanks, I'm on it (good thing the merge conflicted)